src/f32ext.rs

```diff
@@ -61,6 +63,9 @@ pub trait F32Ext: Sized {
     /// Approximate cosine in radians with a maximum error of `0.002`.
     fn cos(self) -> f32;
 
+    /// Calculates Euclidean division, the matching method for `rem_euclid`.
```

Comment:
Can you provide an estimate of the precision / error in this comment? Thank you!

Reply:
The precision should be exactly the same as the std implementation, that is the precision of the `/` operator itself, so I'm not sure exactly what you want me to add in the comment.